plugins/repository-s3/build.gradle

```diff
@@ -167,6 +167,7 @@ if (useFixture) {
     }
   }
 
+  preProcessFixture.dependsOn(composeUp)
```

Comment:
@talevy I think we need this the other way around, `composeUp` needs to run after `writeDockerFile`, that's the missing dependency so that we have the file when run docker compose.

Reply:
thanks!